cassandra-reaper does not launch on docker with cassandra backend

Hi, it seems that the latest docker packaged version does not perform schema update when starting from a fresh db

i just created the db and it s working perfectly
```
ssh 10.46.1.72 nodetool -u cassandra -pw cassandra status reaptest
Datacenter: north
=================
Status=Up/Down
|/ State=Normal/Leaving/Joining/Moving
--  Address     Load       Tokens       Owns (effective)  Host ID                               Rack
UN  10.46.1.88  7.28 GiB   256          100.0%            b29908a2-b28a-442b-90d7-06921b215595  west
UN  10.46.1.72  7.29 GiB   256          100.0%            20e3336a-1484-40bf-8931-e78b80610e64  east
Datacenter: south
=================
Status=Up/Down
|/ State=Normal/Leaving/Joining/Moving
--  Address     Load       Tokens       Owns (effective)  Host ID                               Rack
UN  10.46.1.58  8.73 GiB   256          100.0%            33644628-3c10-4777-a495-fdae4722cfa4  west
UN  10.46.1.97  2.11 GiB   256          100.0%            3afc1744-dad7-4508-8e4d-deb33dec4975  east
```

but when i launch the docker

```docker run -p 8080:8080     -p 8081:8081     -e "REAPER_JMX_AUTH_USERNAME=cassandra"     -e "REAPER_JMX_AUTH_PASSWORD=cassandra"     -e "REAPER_STORAGE_TYPE=cassandra"     -e "REAPER_CASS_CLUSTER_NAME=reaptest"     -e "REAPER_CASS_CONTACT_POINTS=[\"10.46.1.72\"]"     -e "REAPER_CASS_KEYSPACE=reaptest"     thelastpickle/cassandra-reaper:latest```

i got 

```
+ touch /etc/cassandra-reaper.yml
+ /usr/local/bin/configure-persistence.sh
+ /usr/local/bin/configure-webui-authentication.sh
+ /usr/local/bin/configure-metrics.sh
+ exec java -jar /usr/local/lib/cassandra-reaper.jar server /etc/cassandra-reaper.yml
INFO   [2020-01-09 14:03:46,867] [main] i.d.s.DefaultServerFactory - Registering jersey handler with root path prefix: / 
INFO   [2020-01-09 14:03:46,870] [main] i.d.s.DefaultServerFactory - Registering admin handler with root path prefix: / 
INFO   [2020-01-09 14:03:46,870] [main] i.d.a.AssetsBundle - Registering AssetBundle with name: assets for path /webui/* 
INFO   [2020-01-09 14:03:46,948] [main] o.a.s.c.ReflectionBuilder - An instance with name 'authc' already exists.  Redefining this object as a new instance of type org.apache.shiro.web.filter.authc.PassThruAuthenticationFilter 
INFO   [2020-01-09 14:03:46,949] [main] o.a.s.c.ReflectionBuilder - An instance with name 'rest' already exists.  Redefining this object as a new instance of type io.cassandrareaper.resources.auth.RestPermissionsFilter 
INFO   [2020-01-09 14:03:46,992] [main] i.c.ReaperApplication - initializing runner thread pool with 15 threads 
INFO   [2020-01-09 14:03:46,992] [main] i.c.ReaperApplication - initializing storage of type: cassandra 
INFO   [2020-01-09 14:03:47,018] [main] c.d.d.core - DataStax Java driver 3.7.1 for Apache Cassandra 
INFO   [2020-01-09 14:03:47,020] [main] c.d.d.c.GuavaCompatibility - Detected Guava >= 19 in the classpath, using modern compatibility layer 
INFO   [2020-01-09 14:03:47,172] [main] c.d.d.c.ClockFactory - Using native clock to generate timestamps. 
INFO   [2020-01-09 14:03:47,243] [main] c.d.d.c.NettyUtil - Did not find Netty's native epoll transport in the classpath, defaulting to NIO. 
INFO   [2020-01-09 14:03:47,902] [main] c.d.d.c.p.DCAwareRoundRobinPolicy - Using data-center name 'north' for DCAwareRoundRobinPolicy (if this is incorrect, please provide the correct datacenter name with DCAwareRoundRobinPolicy constructor) 
INFO   [2020-01-09 14:03:47,903] [main] c.d.d.c.Cluster - New Cassandra host /10.46.1.58:9042 added 
INFO   [2020-01-09 14:03:47,904] [main] c.d.d.c.Cluster - New Cassandra host /10.46.1.72:9042 added 
INFO   [2020-01-09 14:03:47,904] [main] c.d.d.c.Cluster - New Cassandra host /10.46.1.88:9042 added 
INFO   [2020-01-09 14:03:47,904] [main] c.d.d.c.Cluster - New Cassandra host /10.46.1.97:9042 added 
INFO   [2020-01-09 14:03:48,074] [main] o.c.c.m.MigrationRepository - Found 8 migration scripts 
WARN   [2020-01-09 14:03:48,074] [main] i.c.s.CassandraStorage - Starting db migration from 16 to 23… 
ERROR  [2020-01-09 14:03:48,081] [main] i.c.ReaperApplication - Storage is not ready yet, trying again to connect shortly... 
com.datastax.driver.core.exceptions.InvalidQueryException: unconfigured table running_reapers
	at com.datastax.driver.core.exceptions.InvalidQueryException.copy(InvalidQueryException.java:49)
	at com.datastax.driver.core.DriverThrowables.propagateCause(DriverThrowables.java:35)
	at com.datastax.driver.core.DefaultResultSetFuture.getUninterruptibly(DefaultResultSetFuture.java:293)
	at com.datastax.driver.core.AbstractSession.execute(AbstractSession.java:58)
	at com.datastax.driver.core.AbstractSession.execute(AbstractSession.java:39)
	at io.cassandrareaper.storage.CassandraStorage.initializeAndUpgradeSchema(CassandraStorage.java:254)
	at io.cassandrareaper.storage.CassandraStorage.<init>(CassandraStorage.java:227)
	at io.cassandrareaper.ReaperApplication.initializeStorage(ReaperApplication.java:439)
	at io.cassandrareaper.ReaperApplication.tryInitializeStorage(ReaperApplication.java:310)
	at io.cassandrareaper.ReaperApplication.run(ReaperApplication.java:172)
	at io.cassandrareaper.ReaperApplication.run(ReaperApplication.java:91)
	at io.dropwizard.cli.EnvironmentCommand.run(EnvironmentCommand.java:43)
	at io.dropwizard.cli.ConfiguredCommand.run(ConfiguredCommand.java:87)
	at io.dropwizard.cli.Cli.run(Cli.java:78)
	at io.dropwizard.Application.run(Application.java:93)
	at io.cassandrareaper.ReaperApplication.main(ReaperApplication.java:110)
Caused by: com.datastax.driver.core.exceptions.InvalidQueryException: unconfigured table running_reapers
	at com.datastax.driver.core.Responses$Error.asException(Responses.java:181)
	at com.datastax.driver.core.DefaultResultSetFuture.onSet(DefaultResultSetFuture.java:215)
	at com.datastax.driver.core.RequestHandler.setFinalResult(RequestHandler.java:228)
	at com.datastax.driver.core.RequestHandler.access$2600(RequestHandler.java:62)
	at com.datastax.driver.core.RequestHandler$SpeculativeExecution.setFinalResult(RequestHandler.java:1005)
	at com.datastax.driver.core.RequestHandler$SpeculativeExecution.onSet(RequestHandler.java:808)
	at com.datastax.driver.core.Connection$Dispatcher.channelRead0(Connection.java:1233)
	at com.datastax.driver.core.Connection$Dispatcher.channelRead0(Connection.java:1151)
	at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:105)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:356)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:342)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:335)
	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:286)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:356)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:342)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:335)
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:102)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:356)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:342)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:335)
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:312)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:286)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:356)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:342)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:335)
	at io.netty.channel.ChannelInboundHandlerAdapter.channelRead(ChannelInboundHandlerAdapter.java:86)
	at com.datastax.driver.core.InboundTrafficMeter.channelRead(InboundTrafficMeter.java:38)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:356)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:342)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:335)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1304)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:356)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:342)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:921)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:135)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:646)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:581)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:498)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:460)
	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:131)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)```

it seems that cassandra reaper is accessing the keyspace but he is unable to perform schema update, any clues?

--- a/src/server/src/main/resources/db/cassandra/016_init_reaper_db.cql
+++ b/src/server/src/main/resources/db/cassandra/016_init_reaper_db.cql
@@ -14,7 +14,7 @@
 --  limitations under the License.
 --
 
-CREATE TABLE IF NOT EXISTS reaper_db.running_reapers (
+CREATE TABLE IF NOT EXISTS running_reapers (
     reaper_instance_id uuid PRIMARY KEY,
     last_heartbeat timestamp,
     reaper_instance_host text
@@ -23,7 +23,7 @@ CREATE TABLE IF NOT EXISTS reaper_db.running_reapers (
     AND default_time_to_live = 180
     AND gc_grace_seconds = 180;
 
-CREATE TABLE IF NOT EXISTS reaper_db.repair_unit_v1 (
+CREATE TABLE IF NOT EXISTS repair_unit_v1 (
     id timeuuid PRIMARY KEY,
     blacklisted_tables set<text>,
     cluster_name text,
@@ -38,7 +38,7 @@ CREATE TABLE IF NOT EXISTS reaper_db.repair_unit_v1 (
     AND default_time_to_live = 0
     AND gc_grace_seconds = 864000;
 
-CREATE TABLE IF NOT EXISTS reaper_db.repair_schedule_by_cluster_and_keyspace (
+CREATE TABLE IF NOT EXISTS repair_schedule_by_cluster_and_keyspace (
     cluster_name text,
     keyspace_name text,
     repair_schedule_id timeuuid,
@@ -49,7 +49,7 @@ CREATE TABLE IF NOT EXISTS reaper_db.repair_schedule_by_cluster_and_keyspace (
     AND default_time_to_live = 0
     AND gc_grace_seconds = 864000;
 
-CREATE TABLE IF NOT EXISTS reaper_db.repair_run_by_cluster (
+CREATE TABLE IF NOT EXISTS repair_run_by_cluster (
     cluster_name text,
     id timeuuid,
     PRIMARY KEY (cluster_name, id)
@@ -59,7 +59,7 @@ CREATE TABLE IF NOT EXISTS reaper_db.repair_run_by_cluster (
     AND default_time_to_live = 0
     AND gc_grace_seconds = 864000;
 
-CREATE TABLE IF NOT EXISTS reaper_db.repair_schedule_v1 (
+CREATE TABLE IF NOT EXISTS repair_schedule_v1 (
     id timeuuid PRIMARY KEY,
     creation_time timestamp,
     days_between int,
@@ -78,7 +78,7 @@ CREATE TABLE IF NOT EXISTS reaper_db.repair_schedule_v1 (
     AND default_time_to_live = 0
     AND gc_grace_seconds = 864000;
 
-CREATE TABLE IF NOT EXISTS reaper_db.cluster (
+CREATE TABLE IF NOT EXISTS cluster (
     name text PRIMARY KEY,
     partitioner text,
     seed_hosts set<text>
@@ -88,7 +88,7 @@ CREATE TABLE IF NOT EXISTS reaper_db.cluster (
     AND default_time_to_live = 0
     AND gc_grace_seconds = 864000;
 
-CREATE TABLE IF NOT EXISTS reaper_db.snapshot (
+CREATE TABLE IF NOT EXISTS snapshot (
     cluster text,
     snapshot_name text,
     cause text,
@@ -101,7 +101,7 @@ CREATE TABLE IF NOT EXISTS reaper_db.snapshot (
     AND default_time_to_live = 0
     AND gc_grace_seconds = 864000;
 
-CREATE TABLE IF NOT EXISTS reaper_db.node_metrics_v1 (
+CREATE TABLE IF NOT EXISTS node_metrics_v1 (
     run_id uuid,
     time_partition bigint,
     node text,
@@ -119,7 +119,7 @@ CREATE TABLE IF NOT EXISTS reaper_db.node_metrics_v1 (
     AND default_time_to_live = 180
     AND gc_grace_seconds = 120;
 
-CREATE TABLE IF NOT EXISTS reaper_db.repair_run (
+CREATE TABLE IF NOT EXISTS repair_run (
     id timeuuid,
     segment_id timeuuid,
     cause text static,
@@ -151,7 +151,7 @@ CREATE TABLE IF NOT EXISTS reaper_db.repair_run (
     AND default_time_to_live = 0
     AND gc_grace_seconds = 864000;
 
-CREATE TABLE IF NOT EXISTS reaper_db.repair_run_by_unit (
+CREATE TABLE IF NOT EXISTS repair_run_by_unit (
     repair_unit_id timeuuid,
     id timeuuid,
     PRIMARY KEY (repair_unit_id, id)
@@ -162,7 +162,7 @@ CREATE TABLE IF NOT EXISTS reaper_db.repair_run_by_unit (
     AND default_time_to_live = 0
     AND gc_grace_seconds = 864000;
 
-CREATE TABLE IF NOT EXISTS reaper_db.leader (
+CREATE TABLE IF NOT EXISTS leader (
     leader_id timeuuid PRIMARY KEY,
     last_heartbeat timestamp,
     reaper_instance_host text,
@@ -172,4 +172,3 @@ CREATE TABLE IF NOT EXISTS reaper_db.leader (
     AND compaction = {'class': 'org.apache.cassandra.db.compaction.LeveledCompactionStrategy'}
     AND default_time_to_live = 600
     AND gc_grace_seconds = 600;
-

